sprotty-elk/src/elk-layout.ts

```diff
@@ -182,29 +182,30 @@ export class ElkLayoutEngine implements IModelLayoutEngine {
                 }
             }
         }
-        if (elkNode.labels) {
```

Comment:
Is there a reason, why this is removed? After this change, we would omit labels that are direct children of graphs. I guess this is a very unlikely case, but I just wanted to double-check why it is necessary. Thanks!

Reply:
I moved the code to `applyShape` to avoid duplicating it. I don't think labels as direct children of a graph makes any sense (if you want something like floating comments, you could model those as nodes).

Reply:
Yes, makes sense! Thanks for clarifying!